homeassistant/components/plugwise/coordinator.py

```diff
@@ -65,13 +56,14 @@ async def _connect(self) -> None:
         """Connect to the Plugwise Smile."""
         self._connected = await self.api.connect()
         self.api.get_all_devices()
-        self.name = self.api.smile_name
         self.update_interval = DEFAULT_SCAN_INTERVAL.get(
             str(self.api.smile_type), timedelta(seconds=60)
         )
 
     async def _async_update_data(self) -> PlugwiseData:
         """Fetch data from Plugwise."""
+        data = PlugwiseData(gateway={}, devices={})
+
```

Comment:
Why are you declaring this? Every except raises an issue so data is expected to be set as far as I can see

Reply:
Either being overly cautious but most likely a left over, thanks for spotting.